iwf.yaml

```diff
@@ -1476,3 +1476,30 @@ components:
           format: int64
         attempt: # it's from Attempt of Cadence/Temporal activity.GetInfo
           type: integer
+    EventType:
+      type: string
+      enum:
+        - WORKFLOW_START_EVENT
+        - WORKFLOW_COMPLETE_EVENT
+        - SET_DATA_ATTRIBUTES
```

Comment:
Oh sorry I made this up in the doc for discussion. We don’t need them yet.
But we need other events like state api start, state api complete, state api execution attempt  etc

Reply:
I want to add even a subset of them, for now, to be able to use this new type, `IwfEvent`. Let's discuss what we need